src/CMakeLists.txt

```diff
@@ -306,6 +292,7 @@ add_library(server STATIC ${server_sources})
 add_dependencies(server build_bundled_libs)
 target_link_libraries(server core coll http_parser bit uri swim swim_udp
                       swim_ev crypto mpstream crc32 tzcode)
+target_include_directories(server PUBLIC ${EXTRA_LUA_INCLUDE_DIRS})
```

Comment:
> Why target_include_directories?

Because it is better to limit a scope by target not by directory. libraries above are applied to a target only, not to a directory, it is better to use `target_include_directories` for consistency.

> Why PUBLIC?

changed to `PRIVATE`